src/Microsoft.Repl.Tests/ConsoleHandling/AnsiColorExtensionsTests.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Repl.ConsoleHandling;
+using Xunit;
+
+namespace Microsoft.Repl.Tests.ConsoleHandling
+{
+    public class AnsiColorExtensionsTests
+    {
+        [Theory]
+        [InlineData("BlackText", AllowedColors.BoldBlack, "\x1B[30m\x1B[1mBlackText\x1B[22m\x1B[39m")]
+        [InlineData("RedText", AllowedColors.BoldRed, "\x1B[31m\x1B[1mRedText\x1B[22m\x1B[39m")]
+        [InlineData("GreenText", AllowedColors.BoldGreen, "\x1B[32m\x1B[1mGreenText\x1B[22m\x1B[39m")]
+        [InlineData("YellowText", AllowedColors.BoldYellow, "\x1B[33m\x1B[1mYellowText\x1B[22m\x1B[39m")]
+        [InlineData("BlueText", AllowedColors.BoldBlue, "\x1B[34m\x1B[1mBlueText\x1B[22m\x1B[39m")]
+        [InlineData("MagentaText", AllowedColors.BoldMagenta, "\x1B[35m\x1B[1mMagentaText\x1B[22m\x1B[39m")]
+        [InlineData("CyanText", AllowedColors.BoldCyan, "\x1B[36m\x1B[1mCyanText\x1B[22m\x1B[39m")]
+        [InlineData("WhiteText", AllowedColors.BoldWhite, "\x1B[37m\x1B[1mWhiteText\x1B[22m\x1B[39m")]
+        public void SetColor_WithBold_Both(string text, AllowedColors color, string expected)
```

Comment:
nit: Can we update this and below test names? From the last part, it's not very clear what the expected output is.

Reply:
Done in 4a073b8.